python/kserve/kserve/model_server.py

```diff
@@ -231,6 +233,27 @@ async def stop(self, sig: Optional[int] = None):
             logger.info("Stopping the grpc server")
             await self._grpc_server.stop(sig)
 
+    def add_exception_handler(self, func: Callable[[asyncio.events.AbstractEventLoop, Dict[str, Any]], None]):
```

Comment:
Consider renaming it to `register_exception_handler` for consistency?

Reply:
Renamed